src/meta/SchemaManager.h

```diff
@@ -49,89 +49,18 @@ class SchemaManager {
     virtual SchemaVer getNewestEdgeSchemaVer(folly::StringPiece spaceName,
                                              folly::StringPiece typeName) = 0;
 
-    virtual GraphSpaceID toGraphSpaceID(folly::StringPiece spaceName) = 0;
+    virtual StatusOr<GraphSpaceID> toGraphSpaceID(folly::StringPiece spaceName) = 0;
 
     virtual TagID toTagID(GraphSpaceID space, folly::StringPiece tagName) = 0;
```

Comment:
toTagID  and  toEdgeType  needs to return StatusOr too.

Reply:
It's used within GoExecutor, and GoExecutor doesn't do the exception handling. When it completes the exception handling process, modify it.

Reply:
That's ok.  Change the interface firstly.  Other modules need to depend on it